app/Models/User.php

```diff
@@ -100,4 +100,16 @@ public function isAdmin()
     {
         return $this->role == Role::$admin;
     }
+
+    /**
+     * Checks if the user has accepted the latest privacy policy.
+     * 
+     * @return boolean
+     */
+    public function hasAcceptedPrivacyPolicy()
+    {
+        $acceptedDate = new DateTime($this->privacy_policy_accepted_date);
+        $latestUpdate = new DateTime('2018-05-20');
```

Comment:
I need to change `DateTime` to `\DateTime` so that I don't get an exception. Did that work without backslash on your machine?

Reply:
Ah, I forgot to commit this.